peakinvestigator/internal/logger.py

```diff
@@ -0,0 +1,32 @@
+## -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+import logging
+import time
+
+logging.info('Progress')
+class LoggerProgress(object):
```

Comment:
Shouldn't you be inheriting from `Progress`? Where is it's corresponding `ProgressFactory` implementation?